app/controllers/reviews_controller.rb

```diff
@@ -44,8 +44,10 @@ def destroy
     @review = user_review
     @game = @review.game
     @review.destroy
-    @review.game.update_average
-    redirect_to game_path(@game)
+    unless @game.id.nil?
```

Comment:
When we're in the process of deleting the game. Not sure if we actually need that line, the bug got fixed after a later change.